CONTRIBUTORS.md

```diff
@@ -9,6 +9,8 @@ Thanks to the contributors who helped on this project apart from the authors
 * [Bhargav Sangars](https://www.linkedin.com/in/bhargav-sangars-a4b61037/)
 * [Brend Braeckmans](https://www.linkedin.com/in/brendbraeckmans/)
 * [Rebecca Raj Shree](https://www.linkedin.com/in/rebecca-raj-shree/)
+* [Brent (Johnson) Spetner ](https://www.linkedin.com/in/brentjohnsoneng/)
```

Comment:
An unnecessary whitespace (before the closing bracket)

Reply:
Ah, thanks for the correction. Removed the space 👍